docs/resources/azurerm_public_ip.md

```diff
@@ -0,0 +1,131 @@
+---
+title: About the azurerm_public_ip Resource
+platform: azure
+---
+
+# azurerm\_public\_ip
+
+Use the `azurerm_public_ip` InSpec audit resource to test properties of an Azure Public IP address.
+
+<br />
+
+## Azure REST API version
+
+This resource interacts with version `2020-05-01` of the Azure Management API.
+For more information see the [official Azure documentation](https://docs.microsoft.com/en-us/rest/api/virtualnetwork/publicipaddresses/get).
+
+At the moment, there doesn't appear to be a way to select the version of the
+Azure API docs. If you notice a newer version being referenced in the official
+documentation please open an issue or submit a pull request using the updated
+version.
+
+## Availability
+
+### Installation
+
+This resource is available in the `inspec-azure` [resource
+pack](https://www.inspec.io/docs/reference/glossary/#resource-pack). To use it, add the
+following to your `inspec.yml` in your top-level profile:
+
+    depends:
+      - name: inspec-azure
+        git: https://github.com/inspec/inspec-azure.git
+
+You'll also need to setup your Azure credentials; see the resource pack
+[README](https://github.com/inspec/inspec-azure#inspec-for-azure).
+
+## Syntax
+
+An `azurerm_public_ip` resource block identifies a public IP address by name and Resource Group.
+
+    describe azurerm_public_ip(resource_group: 'example', name: 'addressName') do
+      ...
+    end
+
+<br />
+
+## Examples
+
+### Test that an example Resource Group has the specified IP address
+
+    describe azurerm_public_ip(resource_group: 'example', name: 'public_ip_address') do
+      it { should exist }
+    end
+
+### Test that a specified IP address IP is correct 
+
+    describe azurerm_public_ip(resource_group: 'example', name: 'ClusterName') do
+      its('properties.ipAddress') { should cmp '51.224.11.75' }
+    end
+
+<br />
+
+## Parameters
+
+  - `name`
+  - `resource_group`
+
+## Parameter Examples
+
+The Resource Group as well as the IP address name.
+
+    describe azurerm_public_ip(resource_group: 'example', name: 'public_ip_address') do
+      it { should exist }
+    end
+
+## Attributes
+
+All of the attributes are avialable via dot notation. This is an example of the currently available attributes.
+
+```
+control 'azurerm_public_ip' do
+  describe azurerm_public_ip(resource_group: 'example', name: 'address_name') do
+    it { should exist }
+    its('location') { should cmp 'westeurope' }
+    its('properties.publicIPAddressVersion') { should eq 'IPv4' }
+    its('properties.sku.name') { should eq 'Standard' }
```

Comment:
This should be:
`its('sku.name') { should eq 'Standard' }`

Reply:
@omerdemirok updated